### PR TITLE
bstr-wrap the tokens in the EAT collection

### LINKED ISSUE #23: sub-tokens are bstr-wrapped
In the previous published version of the RMM spec, the sub-tokens were "naked".

In the currently released version of the RMM spec, the sub-tokens are bstr-wrapped.

The encoding and decoding paths need to be updated to reflect the change.

### PR DESCRIPTION
**Pull request by thomas-fossati**
bstr-wrap the tokens in the EAT collection as per latest RMM spec.

Also make sure the resulting code successfully verifies evidence generated by TF-RMM.

Fix #23

### CODE REVIEW

File: testvectors/tf-rmm/cca_platform.pem [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-----BEGIN EC PRIVATE KEY-----
```
Comment (yogeshbdeshpande) [inline testvectors/tf-rmm/cca_platform.pem:1]: Nice!
Can you please also add associated EC public Key. This will be used by me in TA provisioning.

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM! 
Left one minor request!

File: testvectors/tf-rmm/cca_platform.pem (hunk 1 above)
Comment (thomas-fossati) [reply testvectors/tf-rmm/cca_platform.pem:1]: No need to stash it there, one can just:
```bash
openssl ec -in cca_platform.pem -pubout -out cca_platform.pub
```

Comment (yogeshbdeshpande) [reply testvectors/tf-rmm/cca_platform.pem:1]: I  understand that user can search and use commands from internet to extract the corresponding public key.

My suggestion was more from a direction of convenience and consistency of information.
It is far more easy if we put tried and tested valid Private-Public Key pair in the repo for ease of use when working towards new fixes or enhancement.

Comment (thomas-fossati) [reply testvectors/tf-rmm/cca_platform.pem:1]: Whilst I am still unsure about the usefulness of this, I've added it.

Comment (yogeshbdeshpande) [reply testvectors/tf-rmm/cca_platform.pem:1]: Thanks! It is and will be helpful, given the fact we have retained the key pair is a very convenient option!

**Pull request merged**